src/deadline/client/ui/widgets/shared_job_settings_tab.py

```diff
@@ -355,6 +390,15 @@ def get_parameters(self):
                 "value": self.max_retries_per_task_box.value(),
             },
             {"name": "deadline:priority", "type": "INT", "value": self.priority_box.value()},
+            {
+                "name": "deadline:maxWorkerCount",
+                "type": "INT",
+                "value": (
+                    -1
```

Comment:
Instead of populating it with a -1 value, I think we should exclude the item entirely from the list in this case. That will keep better compatibility of job bundles it creates for deadline-cloud library versions that don't have this feature included yet.

Reply:
This does sound like a good idea. So if the unlimited max worker count box is checked, we simply don't populate the -1 at all and since its anyway the default - we should be good with auto-selecting the 'No max worker count' toggle by default when someone imports the job bundle.

Reply:
Yes, think about the case where an existing job bundle is submitted - the behavior should not change with the new code. That means if it's not specified, "No max worker count" must be selected. Then the other direction is if you update the version but are not using the feature, the job bundles it creates are the same as before. That means to exclude the new property name.